doc-content/enterprise-only/admin-and-config/manage-log-file-proc.adoc

```diff
@@ -1,16 +1,22 @@
 [id='manage-log-file-proc']
-= Managing log files
+= Managing log data
 
-{PRODUCT} manages the required maintenance, runtime data that is removed, including:
+{PRODUCT} manages the required maintenance runtime data. It removes some data automatically, including the following data types:
 
 * *Process instance data*, which is removed upon process instance completion.
 * *Work item data*, which is removed upon work item completion.
 * *Task instance data*, which is removed upon completion of a process to which the given task belongs.
 
-Runtime data, which is not cleaned automatically includes session information data that is based on the selected runtime strategy.
+Runtime data that is not cleaned automatically includes session information data that is based on the selected runtime strategy.
 
 * *Singleton strategy* ensures that runtime data of session information is not automatically removed.
 * *Per request strategy* allows automatic removal when a request is terminated.
-* *Per process instances* are automatically removed when a process instance is mapped to a session that is completed or aborted.
+* *Per process instances* automatically removes process instance data when a process instance is mapped to a session that is completed or aborted.
```

Comment:
```suggestion
* *Per process instances* automatically remove process instance data when a process instance is mapped to a session that is completed or aborted.
```

Reply:
The setting (singular) removes the data, not the "instances" (plural)